laravel

```diff
@@ -7,7 +7,7 @@ if (file_exists(__DIR__.'/../../autoload.php')) {
     require __DIR__.'/vendor/autoload.php';
 }
 
-$app = new Symfony\Component\Console\Application('Laravel Installer', '1.2.1');
+$app = new Symfony\Component\Console\Application('Laravel Installer', '1.2.2');
```

Comment:
No worries. I don't there are any guidelines for this tool. It is common that the maintainers handle both versions and changelogs. We don't know if they want to add more stuff before the next release. We also don't know if this is going to be a major update or not.